docs/_components/form-fields.md

```diff
@@ -256,6 +268,18 @@ Three styles of checkboxes are provided as well.
       <input id="c288" class="usa-checkbox__input usa-checkbox__input--tile" type="checkbox" name="c288">
       <label for="c288" class="usa-checkbox__label">Option - Unchecked<span class="usa-checkbox__label-description">Support text</span></label>
     </li>
+    <li>
+      <input id="abc123" class="usa-checkbox__input usa-checkbox__input--tile" type="checkbox" name="abc123">
+        <label for="abc123" class="usa-checkbox__label usa-checkbox__label--illustrated">
```

Comment:
Can we make these IDs unique from the previous example, since I expect there may be accessibility errors with how we have two sets of `abc123` inputs/labels.